### PR TITLE
Resolve gain selection issue #146

### LINKED ISSUE #146: Gain Selection done in ctapipe
See
https://github.com/cta-observatory/ctapipe/pull/1095

I'll try to fix this asap.

### PR DESCRIPTION
**Pull request by vuillaut**
Resolve #146

### CODE REVIEW

File: lstchain/calib/camera/calib.py [hunk 1]
```diff
@@ -48,10 +46,14 @@ def lst_calibration(event, telescope_id):
     dc2pe = event.mc.tel[telescope_id].dc_to_pe  # numgains * numpixels
     signals *= dc2pe
 
-    event.dl1.tel[telescope_id].image = signals
+    threshold = 4094
```
Comment (rlopezcoto) [inline lstchain/calib/camera/calib.py]: we had this threshold defined somewhere else (I think it was for the previous gain_selection...), but we could maybe define it globally for the LST

File: lstchain/calib/camera/calib.py [hunk 2]
```diff
@@ -78,7 +80,7 @@ def gain_selection(waveform, charges, pulse_time, threshold):
     return combined_image, combined_pulse_time
 
 
-
+@deprecated('28/06/2019', message='channel selection is now performed at <= R1 calibration level')
```
Comment (rlopezcoto) [inline lstchain/calib/camera/calib.py]: not sure if we want to leave this here or get rid of it to clean up

File: lstchain/calib/camera/tests/test_calib.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import numpy as np
-from lstchain.calib import camera
+from lstchain.calib.camera.calib import gain_selection
+from astropy.utils import deprecated
 
 
+@deprecated('28/06/2019', message='gain selection is now performed at <= R1 calibration level')
```
Comment (rlopezcoto) [inline lstchain/calib/camera/tests/test_calib.py]: I would at least eliminate the tests

File: lstchain/calib/camera/calib.py (hunk 1 above)
Comment (vuillaut) [reply lstchain/calib/camera/calib.py]: See PR #142 for global definition ;-)

File: lstchain/calib/camera/calib.py (hunk 2 above)
Comment (vuillaut) [reply lstchain/calib/camera/calib.py]: I wanted to leave it until a release that might happen with ctapipe next release (hopefully) and for lst custom calibration

File: lstchain/calib/camera/tests/test_calib.py (hunk 3 above)
Comment (vuillaut) [reply lstchain/calib/camera/tests/test_calib.py]: The function is still used in the lst custom calibration code (that I don't know if some people use it and would anyway leave it until a proper release)

File: lstchain/calib/camera/calib.py (hunk 1 above)
Comment (rlopezcoto) [reply lstchain/calib/camera/calib.py]: Great! :D
The checks are failing in that PR because of the problem on this one, right?

Comment (vuillaut) [reply lstchain/calib/camera/calib.py]: yes!

**Comment by vuillaut** [comment]
Are there still changes to make here?

**Comment by rlopezcoto** [comment]
I am ok with the merging

**Pull request merged**